src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,96 @@
+package pages;
+
+import baseEntities.BasePage;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomePage extends BasePage {
+
+    @FindBy(css = ".login")
+    private WebElement signInButton;
+    @FindBy(xpath = "//div[@class ='shopping_cart']/a")
+    private WebElement cartButton;
+    @FindBy(xpath = "//ul[@id ='homefeatured']/li[@class = 'ajax_block_product col-xs-12 col-sm-4 col-md-3 first-in-line first-item-of-tablet-line first-item-of-mobile-line']")
```

Comment:
not optimal locator. it contains styling elements that could be changed in different browsers/resolutions, should be reworked.